Update the helm installation instructions for arm64 architecture

akhilmukkara opened the pull request:
<!--  Thanks for sending a pull request!  Here are some tips for you -->

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Checklist:**
-   [x] Fixes #423 
-   [x] Signed the commit for DCO to be passed
-   [ ] Labelled this PR & related issue with `documentation` tag

ksatchit commented on docs/litmus-probe.md:
```diff
@@ -9,13 +9,14 @@ sidebar_label: Litmus Probe
+- **promProbe:** To execute promql queries and match prometheus metrics for specific criteria
 
-These probes can be used in isolation or in several combinations to achieve the desired checks. While the `httpProbe` & `k8sProbe` are fully declarative in the way they are conceived, the `cmdProbe` expects the user to provide a shell command to implement checks that are highly specific to the application use case.
+These probes can be used in isolation or in several combinations to achieve the desired checks. While the `httpProbe` & `k8sProbe` are fully declarative in the way they are conceived, the `cmdProbe` expects the user to provide a shell command to implement checks that are highly specific to the application use case. `promProbe` expects the user to provide a promql query along with endpoints to che ...
```
```suggestion
These probes can be used in isolation or in several combinations to achieve the desired checks. While the `httpProbe` & `k8sProbe` are fully declarative in the way they are conceived, the `cmdProbe` expects the user to provide a shell command to implement checks that are highly specific to the application use case. `promProbe` expects the user to provide a promql query along with Prometheus service endpoints to check for specific criteria.
```

ksatchit commented on docs/litmus-probe.md:
```diff
@@ -111,6 +112,32 @@ probe:
+
+### promProbe
+
+The `promProbe` allows developers to run prometheus queries and match the resulting output to follow some criteria. The intent behind this probe was to allow users to declare their prometheus hypothesis, run the promql query for the same on certain endpoints and match the output for specified criteria. If it doesn't follow expected criteria, the experiment should failed.
```
```suggestion
The `promProbe` allows users to run Prometheus queries and match the resulting output against specific conditions. The intent behind this probe is to allow users to define metrics-based SLOs in a declarative way and determine the experiment verdict based on its success. The probe runs the query on a Prometheus server defined by the `endpoint`, and checks whether the output satisfies the specified `criteria`.
```

ksatchit commented on docs/litmus-probe.md:
```diff
@@ -111,6 +112,32 @@ probe:
+
+The `promProbe` allows developers to run prometheus queries and match the resulting output to follow some criteria. The intent behind this probe was to allow users to declare their prometheus hypothesis, run the promql query for the same on certain endpoints and match the output for specified criteria. If it doesn't follow expected criteria, the experiment should failed.
+
+The promql query can be provided in `.query` field. But in case of complex query, the configmap can be created with promql query file. The details of configmap can be passed in chaosengine/chaosexperiment. The `.queryPath` should contains `<cm-mountPath>/<fileName>` value. The `promProbe` can be defined at `.spec.experiments[].spec.probe` the path inside the ChaosEngine.
```
```suggestion
The promql query can be provided in the `query` field. In the case of complex queries that span multiple lines, the `queryPath` attribute can be used to provide the link to a file consisting of the query. This file can be made available in the experiment pod via a ConfigMap resource, with the ConfigMap being passed in the [ChaosEngine](https://docs.litmuschaos.io/docs/chaosengine/#experiment-specification) OR the [ChaosExperiment](https://docs.litmuschaos.io/docs/chaosexperiment/#configuration-specification) CR.  
```

ksatchit commented on docs/litmus-probe.md:
```diff
@@ -111,6 +112,32 @@ probe:
+
+The promql query can be provided in `.query` field. But in case of complex query, the configmap can be created with promql query file. The details of configmap can be passed in chaosengine/chaosexperiment. The `.queryPath` should contains `<cm-mountPath>/<fileName>` value. The `promProbe` can be defined at `.spec.experiments[].spec.probe` the path inside the ChaosEngine.
+
+<strong>NOTE:</strong> `query` and `queryPath` both are mutually exclusive.
```
```suggestion
<strong>NOTE:</strong> `query` and `queryPath` are mutually exclusive.
```

akhilmukkara committed:
Update data.js

Signed-off-by: akhilmukkara <akhil.mukkara@gmail.com>

akhilmukkara committed:
Update data.js

Signed-off-by: akhilmukkara <akhil.mukkara@gmail.com>

PriteshKiri committed:
3.22.x release (#412)

Signed-off-by: akhilmukkara <akhil.mukkara@gmail.com>

lakxy committed:
fix: missing periods (#417)

Signed-off-by: codeEvolveZenith345 <suntiwari3495@gmail.com>
Signed-off-by: akhilmukkara <akhil.mukkara@gmail.com>

lakxy committed:
fix: yaml for correct formatting (#420)

Signed-off-by: codeEvolveZenith345 <suntiwari3495@gmail.com>
Signed-off-by: akhilmukkara <akhil.mukkara@gmail.com>

akhilmukkara committed:
Fix the ARM installation instructions

ARM images are actually now available on dockerhub for mongodb. This PR updates the helm install command with the actual repository and image name.

Signed-off-by: akhilmukkara <akhil.mukkara@gmail.com>

PriteshKiri commented:
Hey @akhilmukkara 

Could you please rebase your branch to sync it with the main?

akhilmukkara commented:
Done! I've rebased the branch with the latest master. @PriteshKiri

akhilmukkara commented:
Closing this PR and creating a clean one with only the relevant changes
